platform/darwin/test/MGLImageSourceTests.m

```diff
@@ -0,0 +1,42 @@
+#import <XCTest/XCTest.h>
+
+#import <Mapbox/Mapbox.h>
+
+@interface MGLImageSourceTests : XCTestCase
+
+@end
+
+@implementation MGLImageSourceTests
+
+
+- (void)testMGLImageSourceWithImageURL {
+
+    MGLCoordinateQuad quad = { { 80, 37}, { 80, 39}, { 81, 39}, { 81, 37}};
```

Comment:
These coordinates appear to be listed in BL/BR/TL/TR order, whereas the struct’s fields are declared in TL/BL/BR/TR order. (×2) Using the designated initializer syntax would head off this mistake, but I wonder if we should introduce a runtime check as well. (A runtime check can be tail work, but we should correct this test before merging.)